### PR TITLE
feat: Allow Copyable to accept iconProps opcionally

### LINKED ISSUE #153: Allow Copyable to accept `iconProps`
### Motivation

To increase flexibility and allow user to customize copy icon, we should accept `iconProps` (optional), and include it in Icon

### Usage example

_No response_

### Possible implementations

_No response_

### PR DESCRIPTION
**Pull request by josevitordaltoe**
closes #153

### CODE REVIEW

**Comment by LuizAsFight** [comment]
@josevitordaltoe Thanks for your contribution! 

This PR is more like a `feat`(feature) than a `fix`.
Fix is used for situations when you're fixing a bug.

File: design-system/react/src/components/Copyable/Copyable.stories.tsx [hunk 1]
```diff
@@ -8,8 +11,18 @@ export default {
...
+  <Flex direction="column">
+    <Copyable value="You copied this text">Standard Icon</Copyable>
+    <Copyable
+      value="Another text which you copied"
+      iconProps={{ icon: Icon.is('Airplane'), 'aria-label': 'An Airplane' }}
+    >
+      Different Icon
+    </Copyable>
+  </Flex>
+);
```
Comment (LuizAsFight) [inline design-system/react/src/components/Copyable/Copyable.stories.tsx:24]: intead of doing different version in same `Usage` component, it's better to do other components, for example:

```
export const Usage = () => <Copyable value="You copied this text">Standard Icon</Copyable>;
export const UsageIconProps = () => (
  <Copyable
    value="Another text which you copied"
    iconProps={{ icon: Icon.is('Airplane'), 'aria-label': 'An Airplane' }}
  >
    Different Icon
  </Copyable>
)
```

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
code is ok.. just minor detail in storybook

File: design-system/react/src/components/Copyable/Copyable.stories.tsx (hunk 1 above)
Comment (josevitordaltoe) [reply design-system/react/src/components/Copyable/Copyable.stories.tsx]: @LuizAsFight first of all, thank you for your detailed help,

i made the change that you requested, but the check workflow above still got one error, is this acceptable ?

thanks again.

File: design-system/react/src/components/Copyable/Copyable.stories.tsx [hunk 2]
```diff
@@ -8,7 +10,17 @@ export default {
   },
 };
 
-export const Usage = () => <Copyable value="Some value">Some value</Copyable>;
+export const Usage = () => (
+  <Copyable value="You copied this text">Standard Icon</Copyable>
+);
+export const UsageIconProps = () => (
```
Comment (LuizAsFight) [inline design-system/react/src/components/Copyable/Copyable.stories.tsx:16]:
```suggestion
export const IconProps = () => (
```

File: design-system/react/src/components/Copyable/Copyable.stories.tsx [hunk 3]
```diff
@@ -8,7 +10,17 @@ export default {
...
+export const IconProps = () => (
+  <Copyable
+    value="Another text which you copied"
+    iconProps={{ icon: Icon.is('Airplane'), 'aria-label': 'An Airplane' }}
+  >
+    Different Icon
+  </Copyable>
+);
 
 Usage.parameters = {
```
Comment (LuizAsFight) [inline design-system/react/src/components/Copyable/Copyable.stories.tsx:25]:
```suggestion
IconProps.parameters = {
  layout: 'centered',
};
Usage.parameters = {
```

**Review by LuizAsFight — APPROVED** [review]
LGTM, thanks again for your contribution @josevitordaltoe

**Pull request merged**